### DISCUSSION TITLE
About using organizations github-page

### DISCUSSION THREAD

**Discussion by stdi** [sdy60-2022/.github#33] [Ideas]
How do you find the idea of using organization's github page (**after customizing it!**) in order to link to the **webring**?

**Comment by sotkot** [comment]
Currently, I have proceeded with netlify suggestion for the Webring. If we cannot succeed as a whole, we can also consider this alternative.

**Reply by stdi** [reply]
These are not mutually exclusive, and not a matter of success but only a suggestion

**Reply by sotkot** [reply]
Agreed. Useful suggestion overall.
